internal/core/application/service.go

```diff
@@ -290,20 +290,10 @@ func (s *Service) UnlockNode(ctx context.Context, password string) error {
 	return nil
 }
 
-func (s *Service) Reset(ctx context.Context) error {
-	backup, err := s.settingsRepo.GetSettings(ctx)
-	if err != nil {
-		return err
-	}
-	if err := s.settingsRepo.CleanSettings(ctx); err != nil {
-		return err
-	}
-	if err := s.storeRepo.ConfigStore().CleanData(ctx); err != nil {
-		// nolint:all
-		s.settingsRepo.AddSettings(ctx, *backup)
-		return err
-	}
-
+func (s *Service) ResetWallet(ctx context.Context) error {
```

Comment:
this can be void now (remove the return at the end of the func body):
```suggestion
func (s *Service) ResetWallet(ctx context.Context) {
```